sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1820,10 +1819,74 @@ void testDetermineCharsetFromMessageFromHttpHeader(@TempDir final File dir) thro
         }
     }
 
+    @Test
+    void testDetermineCharsetFromMessageFromHttpHeaderSuppressEncodingErrors(@TempDir final File dir)
+            throws IOException {
+        try (final MessageStorage messageStorage = new MessageStorage(
+                1, true, mock(MessageFactory.class), new HibernateConfigImpl(dir), this.testRunObserver)) {
+            // given
+            assertEquals(0, messageStorage.getMessageEncodingErrorCount());
+
+            final ListMultimap<String, String> headers = ArrayListMultimap.create();
+            headers.put("Content-Type", "application/xml; charset=ISO-8859-13");
+
+            final HttpApplicationInfo applicationInfo = new HttpApplicationInfo(headers, "transactionId", "requestURI");
+            final TransportInfo transportInfo =
+                    new TransportInfo("http", "localhost", 1234, "remotehost", 4567, List.of());
+            final CommunicationContext communicationContext = new CommunicationContext(applicationInfo, transportInfo);
+            final Message message = new Message(
+                    CommunicationLog.Direction.INBOUND,
+                    CommunicationLog.MessageType.REQUEST,
+                    communicationContext,
+                    messageStorage);
+            message.close();
+
+            // when
+            final Charset actualCharset = messageStorage.determineCharsetFromMessage(message);
+            assertEquals(1, messageStorage.getMessageEncodingErrorCount());
+
+            // then
+            assertEquals(Charset.forName("ISO-8859-13"), actualCharset);
+            Mockito.verifyNoInteractions(this.testRunObserver);
+        }
+    }
+
+    @Test
+    void testDetermineCharsetFromMessageFromHttpHeaderSuppressMimeTypeErrors(@TempDir final File dir)
```

Comment:
Change the access modifier to _public_ and add javadoc.

Reply:
Done.